canonical/ubuntu-pro-client
# feat(gcp): Add noble license code for upgrade (CPC-4104)

## Why is this needed?
To ensure users can upgrade properly from noble release day, this change added the GCP noble license code

## Test Steps
Upgrade a noble gcp VM and ensure it works.

```
gcloud compute instances create pro-upgrade-test --image-project ubuntu-os-cloud-devel --image daily-ubuntu-minimal-2404-noble-amd64-v20240410 --zone=us-central1-a
```

And then follow steps here to upgrade: https://documentation.ubuntu.com/gcp/en/latest/google-how-to/gce/upgrade-in-place-from-lts-to-pro/


## Checklist


## Does this PR require extra reviews?
